src/services/authService.ts

```diff
@@ -149,6 +161,14 @@ export const googleAuthService = async (token: string) => {
       )
     }
 
+    if (user.active === false) {
+      throw new CustomError(
+        NOT_ALLOWED,
+        "Account was deactivated, please consult the admin for more information",
+        403,
+      )
+    }
+
```

Comment:
Same here. I'd suggest:
```suggestion
    if (!user.active) {
      throw new CustomError(
        NOT_ALLOWED,
        "Account was deactivated, please consult the admin for more information",
        403,
      )
    }

```